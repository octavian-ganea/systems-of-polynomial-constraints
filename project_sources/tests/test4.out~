Parsing file test4
Parsing succeded. Results:
List of all variables:
Vector(x, y, z)
List of all polynomials:
Vector((1.0 y^2 + (1.0 z^2)) + 1.0 x^2)

List of desired signs : Vector(<)
Analysed polynomials: 
(1.0 y^2 + (1.0 z^2)) + 1.0 x^2
vars: Vector(x, y, z)

Univariate projection (on real axis) contains these polynomials: 
1.0 z^2
4.0 z^2
256.0 z^2
16.0 z^4
1.0 z^4

There are 31 cells :
Map(x -> 0.0, y -> 0.0, z -> -1.0010000000727433)
Map(x -> 0.0, y -> 0.0, z -> -0.0010000000727431611)
Map(x -> 0.0, y -> 0.0, z -> -7.500001448640364E-4)
Map(x -> 0.0, y -> 0.0, z -> -5.000002169849116E-4)
Map(x -> 0.0, y -> 0.0, z -> -2.505005215020106E-4)
Map(x -> 0.0, y -> 0.0, z -> -1.0008260191095398E-6)
Map(x -> 0.0, y -> -1.0000006610284113, z -> -7.507098209977795E-7)
Map(x -> 0.0, y -> -6.610284113257049E-7, z -> -7.507098209977795E-7)
Map(x -> 0.0, y -> 0.9999993389715887, z -> -7.507098209977795E-7)
Map(x -> 0.0, y -> -1.0000008665293467, z -> -5.005936228860193E-7)
Map(x -> 0.0, y -> -8.665293466422452E-7, z -> -5.005936228860193E-7)
Map(x -> 0.0, y -> 0.9999991334706534, z -> -5.005936228860193E-7)
Map(x -> 0.0, y -> -1.0000009602666207, z -> -2.815729521070724E-7)
Map(x -> 0.0, y -> -9.602666207621484E-7, z -> -2.815729521070724E-7)
Map(x -> -1.00000067004161, y -> -6.871667720130798E-7, z -> -2.815729521070724E-7)
Map(x -> -6.700416099137395E-7, y -> -6.871667720130798E-7, z -> -2.815729521070724E-7)
Map(x -> 0.9999993299583901, y -> -6.871667720130798E-7, z -> -2.815729521070724E-7)
Map(x -> -1.000000865628027, y -> -4.140669232640113E-7, z -> -2.815729521070724E-7)
Map(x -> -8.656280270094098E-7, y -> -4.140669232640113E-7, z -> -2.815729521070724E-7)
Map(x -> 0.999999134371973, y -> -4.140669232640113E-7, z -> -2.815729521070724E-7)
Map(x -> 0.0, y -> 0.9999995859330767, z -> -2.815729521070724E-7)
Map(x -> 0.0, y -> -1.0000009981220594, z -> -6.25522813281255E-8)
Map(x -> 0.0, y -> -9.98122059457808E-7, z -> -6.25522813281255E-8)
Map(x -> -1.0000006619297312, y -> -7.471045417132122E-7, z -> -6.25522813281255E-8)
Map(x -> -6.619297313206382E-7, y -> -7.471045417132122E-7, z -> -6.25522813281255E-8)
Map(x -> 0.9999993380702686, y -> -7.471045417132122E-7, z -> -6.25522813281255E-8)
Map(x -> -1.0000008665293467, y -> -4.960870239686163E-7, z -> -6.25522813281255E-8)
Map(x -> -8.665293466986065E-7, y -> -4.960870239686163E-7, z -> -6.25522813281255E-8)
Map(x -> 0.9999991334706533, y -> -4.960870239686163E-7, z -> -6.25522813281255E-8)
Map(x -> 0.0, y -> 0.999999503912976, z -> -6.25522813281255E-8)
Map(x -> 0.0, y -> 0.0, z -> 0.9999999374477186)
====== FINAL RESULT FOR test4 ================================
null
====================================================
