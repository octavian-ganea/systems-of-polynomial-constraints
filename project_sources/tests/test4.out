====================================================
Parsing file test4
Parsing succeded. Results:
List of all variables:
Vector(x, y, z)
List of all polynomials:
Vector((1.0 y^2 + (1.0 z^2)) + 1.0 x^2)

List of desired signs : Vector(>)
Analysed polynomials: 
(1.0 y^2 + (1.0 z^2)) + 1.0 x^2
vars: Vector(x, y, z)

Univariate projection (on real axis) contains these polynomials: 
1.0 z^2
4.0 z^2
256.0 z^2
16.0 z^4
1.0 z^4

There are 7 cells
Map(x -> 0.0, y -> 0.0, z -> -1.0000000621016214)
Map(x -> 0.0, y -> -1.0000004965376839, z -> -6.210162151170791E-8)
Map(x -> -1.0000008660786868, y -> -4.965376838960563E-7, z -> -6.210162151170791E-8)
Map(x -> -8.660786867984969E-7, y -> -4.965376838960563E-7, z -> -6.210162151170791E-8)
Map(x -> 0.9999991339213132, y -> -4.965376838960563E-7, z -> -6.210162151170791E-8)
Map(x -> 0.0, y -> 0.9999995034623161, z -> -6.210162151170791E-8)
Map(x -> 0.0, y -> 0.0, z -> 0.9999999378983785)
====== FINAL RESULT FOR test4 ================================
Map(x -> 0.0, y -> 0.0, z -> -1.0000000621016214)

